src/modules/70-pipeline/components/PipelineSteps/AdvancedSteps/FailureStrategyPanel/FailureStrategyPanel.stories.tsx

```diff
@@ -42,7 +42,7 @@ export const Basic: Story<BasicArgs> = args => {
         })}
         onSubmit={() => void 0}
       >
-        {formik => {
+        {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
        {(formik: unknown) => {
```